.gitignore

```diff
@@ -3,4 +3,4 @@ tmp/*
 .idea
 .DS_Store
 .byebug_history
-
+.history/
```

Comment:
I'm not sure if we're supposed to modify other parts of the project except the YAML files. If this isn't needed it would probably be good to revert this change

Reply:
I added the .history directory to the .gitignore because my VSCode stores the file history of every change I make. I don't want to push those files to the repo

Reply:
There is a .git/info/exclude file that works as a local .gitignore